src/states/node.rs

```diff
@@ -910,11 +906,12 @@ impl Node {
     // Currently we only send this when nodes join and it's only used to add missing members.
     fn send_section_update(&mut self) {
         trace!("{:?} Sending section update", self);
-        let names = self.peer_mgr.routing_table().our_names();
+        // TODO: do we need to clone?
```

Comment:
Good point! We could just pass `self.peer_mgr.routing_table().our_section()` directly into `get_pub_ids` now.